src/status_im/signals/core.cljs

```diff
@@ -16,20 +16,22 @@
   (log/debug "[signals] status-node-started"
              "error" error)
   (if error
-    {:db (-> db
-             (update :multiaccounts/login dissoc :processing)
-             (assoc-in [:multiaccounts/login :error]
-                       ;; NOTE: the only currently known error is
-                       ;; "file is not a database" which occurs
-                       ;; when the user inputs the wrong password
-                       ;; if that is the error that is found
-                       ;; we show the i18n label for wrong password
-                       ;; to the user
-                       ;; in case of an unknown error we show the
-                       ;; error
-                       (if (= error "file is not a database")
-                         (i18n/label :t/wrong-password)
-                         error)))}
+    (merge {:db (-> db
```

Comment:
I would use
 ```
(cond-> {:db ...} 
   (= (:view-id db) ....)
   (assoc :dispatch [:navigate-to :login]))
```
It's probably a bit terser and should be slightly more efficient